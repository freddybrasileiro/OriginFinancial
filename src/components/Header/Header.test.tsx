import * as React from 'react';
import { shallow } from 'enzyme';
import { Header } from './index';

describe('Header', () => {
  it('Should render the Header component', () => {
    expect(shallow(<Header width="xs"/>)).toMatchSnapshot();
  });
});
