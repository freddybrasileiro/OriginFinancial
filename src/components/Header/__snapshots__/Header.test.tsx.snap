// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render the Header component 1`] = `ShallowWrapper {}`;
